mooncake-store/include/ha_helper.h

```diff
@@ -91,6 +92,34 @@ class MasterServiceSupervisor {
         bool rpc_enable_tcp_no_delay = true,
         const std::string& cluster_id = DEFAULT_CLUSTER_ID,
         BufferAllocatorType memory_allocator = BufferAllocatorType::CACHELIB);
+
+    MasterServiceSupervisor(const MasterConfig &master_config)
+        : enable_gc_(master_config.enable_gc),
```

Comment:
Why move this function from .cpp to .h?

Reply:
this function is constructor, so put in hpp is ok?

Reply:
It's fine to place the constructor in either the .cpp or .h file. However, moving it from .cpp to .h is unrelated to the purpose of this PR, so let's keep it unchanged for now.

Reply:
fix